packages/perseus/src/widgets/interactive-graphs/graphs/circle.tsx

```diff
@@ -46,7 +46,7 @@ function MovableCircle(props: {
     onMove: (newCenter: vec.Vector2) => unknown;
 }) {
     const {center, radius, onMove} = props;
-    const {snapStep, hintMode} = useGraphConfig();
+    const {snapStep, hintMode, readOnly} = useGraphConfig();
```

Comment:
Since both `hintMode` and `readOnly` both come from the graph config and are always used together, what do you think about consolidating them? Creating the graph config would look like this:

```tsx
<GraphConfigContext.Provider
  value={{
    // ...
    disableKeyboardInteraction: hintMode || readOnly,
  }}
>
```

And then individual components would just look at `disableKeyboardInteraction`. That way, the `||` logic wouldn't have to be duplicated in every component.

Reply:
I don't recall exactly where the mouse interactions get disabled. If the code that disables them is reading from `useGraphConfig()` to decide what to do, then `disableInteraction` sounds good to me.

Reply:
I'll double check :)

Reply:
Looks like mouse interactions are disabled just below where we set the config. Doesn't look like it's reading from the config, so will stick with `disableKeyboardInteractions`. 
<img width="551" alt="image" src="https://github.com/user-attachments/assets/144c9a34-8267-46d0-85e4-e207d591f97d">

Reply:
Updated! 😁